code/modules/mob/living/carbon/alien/humanoid/humanoid.dm

```diff
@@ -17,6 +17,8 @@
 	var/drooling = 0 //For Neruotoxic spit overlays
 	bodyparts = list(/obj/item/bodypart/chest/alien, /obj/item/bodypart/head/alien, /obj/item/bodypart/l_arm/alien,
 					 /obj/item/bodypart/r_arm/alien, /obj/item/bodypart/r_leg/alien, /obj/item/bodypart/l_leg/alien)
+	var/sprite_changed_for_emote = FALSE
+	var/emote_length = 12

```

Comment:
Oh, that's what he meant. I was half asleep and worried I was being told I was being a numpty and didn't understand.